plugins/open-feedback/src/components/OpenFeedbackModal/OpenFeedbackModal.tsx

```diff
@@ -20,22 +20,26 @@ import {
   FormControlLabel,
   Checkbox,
   DialogActions,
+  Fab,
 } from '@material-ui/core';
+import { cleanUrl } from '../../utils/cleanUrl';
 
-export type SidebarOpenfeedbackProps = {
+export type ButtonOpenfeedbackProps = {
   icon?: IconComponent;
+  floating?: boolean;
+  style?: React.CSSProperties;
```

Comment:
I think we need to do two things here to keep it more customisable for everyone, lets add a `color?` and `title?` to these props like:

```ts
type ButtonOpenfeedbackProps = {
  icon?: IconComponent;
  floating?: boolean;
  style?: React.CSSProperties;
  title?: string;
  color?: 'primary' | 'secondary';
};
```

Then we give people the option to have the modal fit their Backstage app color theme and can give it a fancy title if they like! Then we can just pass it as:

```ts
        <Fab
          color={props.color ?? 'primary'}
          variant="extended"
          onClick={() => setOpen(true)}
          style={props.style}
        >
          <Icon style={{ marginRight: 4 }} />
          {props.title}
        </Fab>
 ```

And of then of course update the readme to say you can do this:

```ts
    <OpenFeedbackModal
      title="Super Feedback!"
      floating
      color='secondary'
      icon={FeedbackIcon}
      style={{ position: 'fixed', bottom: 20, right: 20, color: 'white' }}
    />
```

Hope you don't think I am being too picky! I just think its a really nice to have!

WDYT?

Reply:
Not picky at all! I really appreciate the detailed review and feedback. It’s my pleasure to have my work reviewed by a senior engineer! https://github.com/Parsifal-M/backstage-open-feedback/pull/49/commits/642880f5f94f2238fae66f2edf40876e511d5bef